dev/io.openliberty.org.jboss.resteasy.jaxb.provider.ee10/bnd.bnd

```diff
@@ -0,0 +1,60 @@
+#*******************************************************************************
+# Copyright (c) 2022 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+
+bVersion=1.0
+
+Bundle-SymbolicName: io.openliberty.org.jboss.resteasy.jaxb.provider.ee10
+
+resteasy-version=6.0.0.Final
+
+instrument.ffdc: false
+instrument.disabled: true
+
+app-resources= \
+  META-INF/services/jakarta.ws.rs.ext.Providers
+
+-cdiannotations:
+
+Service-Component: \
+  io.openliberty.org.jboss.resteasy.jaxb.provider.ResourceProvider; \
+    implementation:=com.ibm.wsspi.classloading.ResourceProvider; \
+    provide:=com.ibm.wsspi.classloading.ResourceProvider; \
+    configuration-policy:=optional; \
+    properties:="resources=${app-resources}"
+
+Export-Package: \
+  org.jboss.resteasy.annotations.providers.jaxb;version=${resteasy-version};thread-context=true, \
+  org.jboss.resteasy.plugins.providers.jaxb;version=${resteasy-version};thread-context=true
+ Private-Package: \
+  org.jboss.resteasy.plugins.providers.jaxb.i18n;version=${resteasy-version}
+
+# Using version=! in order to not have a version attached to the import for packages that were removed
+# from Java after Java 8.  Doing this keeps the import like before Java 11 support. It will get the
+# packages from Java when using Java 8 or earlier and from the new shipped bundles for Java 9 and later.
+Import-Package: \
+  jakarta.xml.bind;version=!, \
+  jakarta.xml.bind.annotation;version=!, \
+  jakarta.xml.bind.annotation.adapters;version=!, \
+  jakarta.xml.bind.attachment;version=!, \
+  jakarta.xml.bind.helpers;version=!, \
+  jakarta.xml.bind.util;version=!, \
+  !com.sun.xml.bind.marshaller, \
+  *
+#  jakarta.activation;version=!, \
+#  jakarta.annotation;version=!, \
+#  jakarta.annotation.security;version=!, \
```

Comment:
Do not need to use version=! since it is jakarta instead of javax.  It isn't in the JDK with the jakarta package name.  It is the same with the jakarta.activation and annotation.  If anything you would update it to version=[3.0,5.0) possibly if it is going to be used for both restfulWS 3.0 and 3.1, but since there is no buildpath it is just going to do no version since there is nothing to know what version to import

Reply:
The entire Import-Package can be removed.